EnglishFilter/sections/foreign.txt

```diff
@@ -5466,27 +5466,13 @@ dap-news.com##.ads
 @@||gmyankee.tistory.com^$generichide
 @@||hjkhgyss.tistory.com^$generichide
 @@||liveinincheon.tistory.com^$generichide
-
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=liveinincheon.tistory.com
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=hjkhgyss.tistory.com
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=maniac-game.tistory.com
-
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=viewingcat.tistory.com
+@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=liveinincheon.tistory.com|hjkhgyss.tistory.com|maniac-game.tistory.com
```

Comment:
Is this necessary? I don't see the blocking request. Please provide a screenshot.

Reply:
This is just sorting a bunch of rules.

Reply:
Ok.